src/Core/Routing/ControllerStack.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
@Progi1984 You will tell me to add declare strict type
I trust you 🙏

Reply:
https://devdocs.prestashop.com/1.7/development/coding-standards/#strict-typing